### ISSUE TITLE
How do you specify an autoexec for winiomwin?

### ISSUE THREAD

**Issue by dustinfarris** [sassoftware/saspy#94]
I see how to do it for stdout, but not for any other method.

**Comment by tomweber-sas** [comment]
Since you are connecting to an IOM Workspace server, the autoexec and any other config would already be set up and configured for that workspace server. So, there's no way to pass a parameter through from the client.
Now, if you're using the Local IOM connection, then you have a local SAS install on your same PC that you're using. In that install, you have the sasv9,cfg, like the workspace server does. You configure anything like that using the normal SAS configuration. So, really it's the same as the remote configuration, but you can do it yourself (for remote, it's likely someone else that had configured those SAS sessions).
So, basically the answer is you don't do that with saspy, you do it with SAS. Same as you would do for when you just run sas.exe yourself.

That make sense?
Tom

**Comment by dustinfarris** [comment]
Yeah that makes sense.  My conundrum is that I want this script to be executable by many users, who may or may not have the same autoexec configured.  I was hoping to override it with something consistent.  I think I can get by without this though.  Thanks, as always, for the quick response!

**Comment by tomweber-sas** [comment]
You can simply 
```
log_lst = sas.submit("%include '/path/to/autoexec';")
```
as the first thing you your program. That should have the same effect. Would that work?

Thanks,
Tom

**Comment by dustinfarris** [comment]
That's what I've been doing :D

-- 
Dustin Farris
